scripts/release-helpers/release-gen.sh

```diff
@@ -139,6 +151,18 @@ metadata.annotations.alm-examples: |-
 
 spec.version: $NEW_VERSION
 spec.install.spec.deployments[0].spec.template.spec.containers[0].image: registry.connect.redhat.com/storageos/cluster-operator:$NEW_VERSION
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[0].value: registry.connect.redhat.com/storageos/node:1.5.2
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[1].value: registry.connect.redhat.com/storageos/init:1.0.0
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[2].value: registry.connect.redhat.com/k8scsi/csi-cluster-driver-registrar:v1.0.1
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[3].value: registry.connect.redhat.com/k8scsi/csi-node-driver-registrar:v1.2.0
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[4].value: registry.connect.redhat.com/storageos/csi-provisioner:v1.4.0
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[5].value: registry.connect.redhat.com/k8scsi/csi-attacher:v1.2.1
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[6].value: registry.connect.redhat.com/k8scsi/csi-attacher:v2.0.0
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[7].value: registry.connect.redhat.com/k8scsi/livenessprobe:v1.1.0
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[8].value: registry.connect.redhat.com/k8scsi/driver-registrar:v0.4.2
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[9].value: registry.connect.redhat.com/storageos/csi-provisioner:v0.4.3
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[10].value: registry.connect.redhat.com/k8scsi/csi-attacher:v0.4.2
+spec.install.spec.deployments[0].spec.template.spec.containers[0].env[11].value: registry.connect.redhat.com/storageos/nfs:1.0.0
```

Comment:
These values must be updated once we have the exact image URI from the registry.

Reply:
Do we need to provide defaults here or are they just documentation examples?

Reply:
Usually the CSVs are treated like immutable releases. So, the example in the docs should be the final form of the CSV.
If we don't have all the images in the repo, we can leave them empty for now. Empty string will result in falling back to the defaults images. Using a string template would result in attempts to pull the image.
I could not find any existing operators in operatorhub.io using these env vars for reference.

Reply:
In that case, I think it would be best to leave them empty for now.  I think we should prefer the current CR approach and only use the env vars where required by the marketplace.

Reply:
Okay, I'll leave them empty for now.